src/main/java/duke/command/AddDeadlineTaskCommand.java

```diff
@@ -0,0 +1,44 @@
+package duke.command;
+
+import java.time.LocalDate;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a command that when executed adds a deadline task to the task list.
+ */
+public class AddDeadlineTaskCommand extends Command {
+    private final String description;
+    private final LocalDate date;
+
+    /**
+     * Class constructor.
+     * 
+     * @param description description of the deadline task.
+     * @param date deadline of the task expressed in date.
+     */
+    public AddDeadlineTaskCommand(String description, LocalDate date) {
+        this.description = description;
+        this.date = date;

```

Comment:
Perhaps the Bot could integrate time into Deadline as well?

Reply:
I agree with you. Will implement this in the future.